config/rbac/kustomization.yaml

```diff
@@ -4,7 +4,8 @@ resources:
 # if your manager will use a service account that exists at
 # runtime. Be sure to update RoleBinding and ClusterRoleBinding
 # subjects if changing service account names.
-- service_account.yaml
+- service_account_kube_proxy.yaml
```

Comment:
Will a Kubernetes admin need to do anything special when upgrading to this version of OFO?

Reply:
Ya and after a release we can do some smoke testing as well.